fenics_ice/inout.py

```diff
@@ -306,37 +306,85 @@ def field_from_vel_file(infile, field_name):
 
 def read_vel_obs(params, model=None):
```

Comment:
Replace `params` with a list of arguments, `infile`and `use_cloud_point_velocities` (the latter default `False`). This will let us read multiple inputs.